backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/helper/ImplicitSpecializationSwitch.java

```diff
@@ -661,6 +661,18 @@ public List<Specialization> caseFeature(Feature object) {
         if (!this.hasRedefinition(object)) {
             implicitSpecializations.addAll(this.handleImplicitParameterRedefinition(object));
         }
+        // The specification states that "If the Feature has chainingFeatures, then the union also includes the types of
```

Comment:
It seems strange to set the "type" property of a Feature here.
Your comment comes from the "deriveFeatureType" section of the KerML specification, so why did you not implement that part in the  "FeatureImpl#getType()" method?
If you have good reasons to do that, please explain it here (in the java file) in a detailed comment.

Reply:
I need to implement this here in order to make inherited `Feaure` resolvable.

If I only implement this in `getType` the general mecanisme of name resolution that relies on `membership` will not work.

In a normal usecase this `implicit` typing would be present in the model (Either with an implicit inheritance or implicit typing). But since in SysON we choose to add those "implicit" element virtually when the derived feature a called, i needed to find a place where this 'implicit' typing would impact both:

* GetType
* GetMembership (to make org.eclipse.syson.sysml.impl.NamespaceImpl.resolveLocal(String) work)